source/nodejs/adaptivecards-ui-testapp/package.json

```diff
@@ -0,0 +1,58 @@
+{
+	"name": "adaptivecards-ui-testapp",
+	"version": "1.0.0",
+	"description": "Harness for UI testing for the Adaptive Cards rendering library",
+	"author": "AdaptiveCards",
+	"license": "MIT",
+	"homepage": "https://adaptivecards.io",
+	"keywords": [
+		"adaptivecards",
+		"adaptive",
+		"cards",
+		"microsoft",
+		"bot"
+	],
+	"main": "lib/adaptivecards-ui-testapp.js",
+	"types": "lib/adaptivecards-ui-testapp.d.ts",
+	"files": [
+		"lib",
+		"dist",
+		"src"
+	],
+	"scripts": {
+		"clean": "rimraf build lib dist",
+		"prebuild": "tsc",
+		"build": "webpack",
+		"watch": "webpack --watch",
+		"start": "webpack-dev-server --open",
+		"dts": "dts-generator --prefix adaptivecards-ui-testapp --project . --out dist/adaptivecards-ui-testapp.d.ts",
+		"lint": "eslint src/**/*.ts",
+		"iterate": "webpack --cache --mode=production --progress && npm run dts",
+		"release": "npm run build && webpack --mode=production && npm run dts"
+	},
+	"dependencies": {
+		"adaptivecards-controls": "^0.9.0-beta.1"
+	},
+	"peerDependencies": {
+		"adaptive-expressions": "^4.11.0",
```

Comment:
These should probably be regular or dev dependencies here, right?

Reply:
you're right, the dependencies are duplicated in peer and dev dependencies